XML/VMConfigurations/TwoVMs.xml

```diff
@@ -394,4 +394,90 @@
             </VirtualMachine>
         </ResourceGroup>
     </TwoVM12Disk>
+    <LPST>
+        <isDeployed>NO</isDeployed>
+        <ResourceGroup>
+            <VirtualMachine>
+                <ARMInstanceSize>Standard_DS15_v2</ARMInstanceSize>
+                <RoleName></RoleName>
```

Comment:
RoleName is missing in both VM

Reply:
The scripts use the default 'role-0'/'role-1' role names